sympy/combinatorics/tensor_can.py

```diff
@@ -40,7 +40,7 @@ def dummy_sgs(dummies, sym, n):
 
     n : number of indices
 
-    in base form the dummy indices are always in consecutive positions
+    In base form the dummy indices are always in consecutive positions.
```

Comment:
```suggestion
    In base form, the dummy indices are always in consecutive positions.
```

Reply:
Also, shouldn't this come under the explanation for dummies?

Reply:
Changed.